packages/react-component-library/src/components/Select/hooks/useSelectMenu.ts

```diff
@@ -0,0 +1,22 @@
+import React, { useCallback } from 'react'
+
+export function useSelectMenu(
+  inputRef: React.RefObject<HTMLInputElement>,
+  closeMenu: () => void
+): {
+  onMenuKeyDownHandler: (e: React.KeyboardEvent<HTMLUListElement>) => void
+} {
+  const onMenuKeyDownHandler = useCallback(
+    (e: React.KeyboardEvent<HTMLUListElement>) => {
+      if (e.key === 'Tab') {
+        inputRef.current?.focus()
+        closeMenu()
```

Comment:
Small thing but I was wondering if the closeMenu() is necessary – was there a problem without it?